WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -200,6 +204,20 @@ extension NotificationDetailsViewController: UITableViewDelegate, UITableViewDat
         return cell
     }
 
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        estimatedRowHeightsCache.setObject(cell.frame.height as AnyObject, forKey: indexPath as AnyObject)
+    }
+
+    func tableView(_ tableView: UITableView, estimatedHeightForRowAt indexPath: IndexPath) -> CGFloat {
+        if let height = estimatedRowHeightsCache.object(forKey: indexPath as AnyObject) as? CGFloat {
+            return height
+        }
+        return Settings.estimatedRowHeight
```

Comment:
So for these, it's actually indeed a visual layout bug. When using autosizing cells, any kind of refresh on cells, or even multiple layout passes, can end up _tweaking_ the heights.

My understanding, `UITableView` does some weird pre-calculation when laying out cells in which they are first set to their estimated height and _then_ set to their computed height. So using the exact computed height, when we have it, as the estimated height prevents the layout tweaking.